db/models.py

```diff
@@ -50,3 +62,75 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
```

Comment:
Ensure that the `AUTH_USER_MODEL` setting in your Django settings is pointing to this custom `User` model. This is essential for the authentication system to work correctly.